releases/nightly-build/compose-files/docker-compose-nexus-mongo-arm64.yml

```diff
@@ -35,6 +34,8 @@ x-common-env-variables: &common-variables
   SecretStore_Host: edgex-vault
   SecretStore_ServerName: edgex-vault
   SecretStore_RootCaCertPath: /tmp/edgex/secrets/ca/ca.pem
+#  Clients_Notifications_Host: edgex-support-notifications # un-comment if re-enabling logging service for remote logging
+#  Logging_EnableRemote: "true" # un-comment if re-enabling logging service for remote logging
```

Comment:
Instead of 

```
#  Logging_EnableRemote: "true" # un-comment if re-enabling logging service for remote logging
```

how about

```
Logging_EnableRemote: "false" # Set to "true" if re-enabling logging service for remote logging
```

Reply:
Never mind. I see this is consistent with the rest of the PR.